packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -143,12 +162,34 @@ class MenuContent extends React.Component {
   }
 
   render() {
+    let index = -1;
     const items = React.Children.map(this.props.children, (item) => {
+      let onClick = item.props.onClick;
+      let newItem = item;
+
       if (item.props.subMenuItems && item.props.subMenuItems.length > 0) {
-        return React.cloneElement(item, { onClick: this.wrapOnClick(item), onKeyDown: this.wrapOnKeyDown(item) });
+        onClick = this.wrapOnClick(item);
+      }
+
+      if (item.props.text) {
+        index += 1;
+        const onKeyDown = this.wrapOnKeyDown(item, index);
+        const isActive = this.state.focusIndex === index;
+
+        newItem = React.cloneElement(item, {
+          onClick,
+          onKeyDown,
+          isActive,
+        });
+      } else if (item.props.children) {
+        const children = React.Children.map(item.props.children, (child) => {
+          index += 1;
+          return React.cloneElement(child, { onKeyDown: this.wrapOnKeyDown(child, index), isActive: index === this.state.focusIndex });
```

Comment:
nit: on line 179 you place each cloneElement prop on a new line and here is same line